Tests/presentations/translators.lisp

```diff
@@ -19,24 +19,36 @@
                 (object)
               (etypecase object
                 (real (format nil "~a" object))
-                (string object))))
-        (all-1 (find-presentation-translators 'real 'string 'pt.smoke-ct2))
-        (all-2 (find-presentation-translators 'string 'string 'pt.smoke-ct2))
-        (all-3 (find-presentation-translators '(or string real) 'string 'pt.smoke-ct2))
-        (all-4 (find-presentation-translators '(or string integer) 'string 'pt.smoke-ct2))
-        (all-5 (find-presentation-translators '(and string (member "a" "b")) 'string 'pt.smoke-ct2))
-        (all-6 (find-presentation-translators '(member "a" "b") 'string 'pt.smoke-ct2))
-        (all-7 (find-presentation-translators '(member "dan" 3) 'string 'pt.smoke-ct2))
-        (non-1 (find-presentation-translators 'number 'string 'pt.smoke-ct2))
-        (non-2 (find-presentation-translators '(or string number) 'string 'pt.smoke-ct2))
-        (non-3 (find-presentation-translators '(member "dan" :foo) 'string 'pt.smoke-ct2)))
```

Comment:
cases with member are also meaningful, so please replace them with expanded versions (that is use completion), instead of removing them.

Reply:
Fixed.